lib/Charts/BarChart.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+import {select as d3Select} from 'd3-selection';
+import d3Sync from './d3Sync';
+
+import defined from 'terriajs-cesium/Source/Core/defined';
+
+import BaseChart from './BaseChart';
+
+import assign from '../Core/object-assign';
+
+const barWidth = 6;
+
+const BarChart = assign({}, BaseChart, {
+    renderChart(renderContext) {
+        const {g, data, scales} = renderContext;
+        
+        // animations can kiss my ass
+        
+        d3Sync(g.selectAll('.line'), data, 'g', (dataline, datalines_count) => {
```

Comment:
haven't come across this way of doing things before - can you explain it to me? It would be nice to have animations consistent with the line chart too...